content/en/guides/hosting/data-security/secure-storage-connector.md

```diff
@@ -108,61 +115,70 @@ For details, see [Create a CoreWeave AI Object Storage bucket](https://docs.core
 
-    ```json
-    {
-      "Version": "2012-10-17",
-      "Statement": [
-      {
-        "Sid": "AllowWandbUser",
-        "Action": [
-          "s3:GetObject*",
-          "s3:GetEncryptionConfiguration",
-          "s3:ListBucket",
-          "s3:ListBucketMultipartUploads",
-          "s3:ListBucketVersions",
-          "s3:AbortMultipartUpload",
-          "s3:DeleteObject",
-          "s3:PutObject",
-          "s3:GetBucketCORS",
-          "s3:GetBucketLocation",
-          "s3:GetBucketVersioning"
-        ],
-        "Effect": "Allow",
-        "Resource": [
-          "arn:aws:s3:::<cw-bucket>/*",
-          "arn:aws:s3:::<cw-bucket>"
-        ],
-        "Principal": {
-          "CW": "arn:aws:iam::wandb:static/wandb-integration-public"
-        },
-        "Condition": {
-          "StringLike": {
-            "wandb:OrgID": [
-              "<wb-org-id>"
-            ]
-          }
-        }
-      },
-      {
-        "Sid": "AllowUsersInOrg",
-        "Action": "s3:*",
-        "Effect": "Allow",
-        "Resource": [
-          "arn:aws:s3:::<cw-bucket>",
-          "arn:aws:s3:::<cw-bucket>/*"
-        ],
-        "Principal": {
-          "CW": "arn:aws:iam::<cw-storage-org-id>:*"
+        ```json
+        {
+          "Version": "2012-10-17",
+          "Statement": [
+          {
+            "Sid": "AllowWandbUser",
+            "Action": [
+              "s3:GetObject*",
+              "s3:GetEncryptionConfiguration",
+              "s3:ListBucket",
+              "s3:ListBucketMultipartUploads",
+              "s3:ListBucketVersions",
+              "s3:AbortMultipartUpload",
+              "s3:DeleteObject",
+              "s3:PutObject",
+              "s3:GetBucketCORS",
+              "s3:GetBucketLocation",
+              "s3:GetBucketVersioning"
+            ],
+            "Effect": "Allow",
+            "Resource": [
+              "arn:aws:s3:::<cw-bucket>/*",
+              "arn:aws:s3:::<cw-bucket>"
+            ],
+            "Principal": {
+              "CW": "arn:aws:iam::wandb:static/wandb-integration-public"
+            },
+            "Condition": {
+              "StringLike": {
+                "wandb:OrgID": [
+                  "<wb-org-id>"
+                ]
+              }
+            }
+          },
+          {
+            "Sid": "AllowUsersInOrg",
+            "Action": "s3:*",
+            "Effect": "Allow",
+            "Resource": [
+              "arn:aws:s3:::<cw-bucket>",
+              "arn:aws:s3:::<cw-bucket>/*"
+            ],
+            "Principal": {
+              "CW": "arn:aws:iam::<cw-storage-org-id>:*"
+            }
+          }]
         }
-      }]
-    }
-    ```
 
-The clause beginning with `"Sid": "AllowUsersInOrg"` grants users in your W&B organization direct access to the bucket. If you don't need this ability, you can omit the clause from your policy.
+        ```
```

Comment:
Heads up the formatting looked off in the preview:
<img width="1139" height="824" alt="Screenshot 2025-08-28 at 17 15 27" src="https://github.com/user-attachments/assets/8397fd30-c956-479d-9140-99c77a34f1bd" />